src/mars_patcher/data/schema.json

```diff
@@ -614,6 +620,17 @@
             "type": "string",
             "enum": ["0", "1", "2", "3", "4", "5", "6"]
         },
+        "ItemMessages": {
+            "type": "object",
+            "propertyNames": {
+                "$ref": "#/$defs/ValidLanguages"
+            },
+            "additionalProperties": {
+                "type": "string",
+                "description": "Specifies what text should appear on acquiring this item. Two lines are available, with an average of 28 characters per line. Use \n to force a line break. If not provided, a message based on the Item will be shown.",
```

Comment:
Curious: what happens if you have something like:
```json
{
  "English": "bla"
}
``` 
What will happen to german/french/spanish/italian/JP? Will those have default messages based on their languages shown?

Reply:
After looking at the assembly looks like everything non-english will just use english.